frontend/main.tsx

```diff
@@ -7,10 +7,18 @@ import {
 import App from './App'
 // import ErrorPage from './pages/error-page'
 import './index.css'
-// import CreateItinerary from './components/CreateItinerary'
-// import Dashboard from './components/Dashboard'
+import CreateAccount from './pages/create-account'
+import ManageAccount from './pages/manage-account'
 
 const router = createBrowserRouter([
+  {
```

Comment:
Here are the paths I was referencing in the App.tsx file comment. We can have the '/' route redirect to, say, '/home', and show that page only if the user is not logged in. But I think we should have the 'Landing' aka the Login page in a separate route, not '/', because of how we're going to be adding the header for other pages.